metal3-dev/local-cbo.sh

```diff
@@ -8,6 +8,21 @@ source $SCRIPTDIR/logging.sh
 source $SCRIPTDIR/common.sh
 source $SCRIPTDIR/utils.sh
 
+# Stop already running CBO.
+pushd $SCRIPTDIR
+$SCRIPTDIR/stop-cbo.sh
+
+CBO_LOCAL_IMAGE_PATH="/etc/cluster-baremetal-operator/images"
+
+# Copy cbo-images.json into the CBO_LOCAL_IMAGE_PATH to run CBO locally. 
+# CBO needs image.json file in this path.
+if [[ ! -f "${CBO_LOCAL_IMAGE_PATH}/images.json" ]]; then
```

Comment:
The operator takes a command line argument to pass this file, so we could avoid cluttering the real system files of the local host by putting it in /opt/dev-scripts or using the version in the OCP_DIR if we could pass that value to `make` on line 42. That may require changes in the cluster-baremetal-operator Makefile to update the `run` target.